src/commands/weather.js

```diff
@@ -1,5 +1,5 @@
 const { checkClearance, makePipe, textContains } = require('./util')
-const rnd = require('../randtools')
+const rnd = require('randtools')
```

Comment:
Is this working? Shouldn't it require `src/randtools`?

Reply:
Good point, fixed!